include/opentxs/network/zeromq/ListenCallback.hpp

```diff
@@ -42,7 +42,7 @@ class OPENTXS_EXPORT ListenCallback
 
     /// Deactivate will block until Process is no longer executing, unless it is
     /// called by the same thread as the one currently executing the callback.
-    virtual auto Deactivate() const noexcept -> void = 0;
```

Comment:
It is not const because the call changes the state of the object. To contain the change, the original used const_cast, which really is reserved for cases where we need to interface to old software or contain the fallout of existing design errors. False const qualification is misleading, it does not serve any useful purpose and it may be implicated in cases where the top level of compiler optimization might cause errors.